pkg/apis/externaldns/types.go

```diff
@@ -285,7 +285,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.DefaultEnvars()
 
 	// Flags related to Kubernetes
-	app.Flag("master", "The Kubernetes API server to connect to (default: auto-detect)").Default(defaultConfig.Master).StringVar(&cfg.Master)
+	app.Flag("apiserver", "The Kubernetes API server to connect to (default: auto-detect)").Default(defaultConfig.APIServerURL).StringVar(&cfg.APIServerURL)
```

Comment:
Let's call it `--server` similar to [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/).

Reply:
Legit, will do the change.